.travis.yml

```diff
@@ -19,3 +19,11 @@ matrix:
     - rvm: jruby-head
     - rvm: rbx-3
   fast_finish: true
+script:
+  # Unit test
+  - bundle exec rake
+  # Install test
+  - gem build rb-inotify.gemspec
+  - gem install rb-inotify-*.gem
+  - "gem list | grep rb-inotify"
+  - ruby -e 'require "rb-inotify"'
```

Comment:
This test is to test `require "rb-inotify"` (Runtime error).
But it would be better that the test may be included in Unit Test.